lib/form-utils.tsx

```
@@ -0,0 +1,226 @@
+    if (field.validation) {
+      const error = field.validation(value)
+      if (error) {
+        errors[field.id] = error
+        return
+      }
+    }
+
+    // Run type-specific validation
+    switch (field.type) {
+      case 'number':
+        const numberError = validators.number(value)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)